modules/kn-trigger-create.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kn-trigger.adoc
+
+[id="kn-trigger-create_{context}"]
+= Creating a trigger using the Knative CLI
+
+.Procedure
+
+* To create a trigger, enter:
++
+[source,terminal]
```

Comment:
Unfortunately, we didn't backport the [source,terminal] formatting back to 4.4. We stopped at 4.5, so all of these source,terminal entries are going to need to be removed throughout your PR. Sorry :( :(

Reply:
Oops lol some of the code blocks already had this, so at least I know now to remove _all_ of them